src/pages/painting/[slug].js

```diff
@@ -39,6 +44,24 @@ export async function getStaticProps({ params }) {
     name: paintingResponse.collection?.name,
   });
 
+// If theres 1 painting in the collection we will remove the current painting from the recommended
+//  list and recommend paintings from a different collection
+  if (collectionResponse.paintingsCollection.items.length <= 1) {
+    const allPaintingsResponse = await getAllPaintings();
+
+    const recommendedPaintings = removeCurrentPaintingFromRecommendedList(
+      paintingResponse.name,
+      allPaintingsResponse
+    );
+
+    return {
+      props: {
+        paintingData: paintingResponse,
+        collectionData: recommendedPaintings.paintingsCollection,
+      },
+    };
+  }
+
   // In case the collection field is empty, the page won't break
   if (!collectionResponse) {
```

Comment:
Do you think this is still necessary? I know I added it in the past, but I wonder if its redundant now.